behat/features/login.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login
+  In order to be able to login to the student personal area
+  As an anonymous user
+  We need to be able to have access to the student personal area
+
+  @javascript
+  Scenario: Visit the login area page
+    Given I am on login page
+    When  I fill in "EmailOrUsername" with "boaz@eteachergroup.com"
+    And   I fill in "Password" with "Zaq12wsx"
```

Comment:
@amitaibu - It's that ok to insert the user name and the password in the scenario? or you prefer that the user name and password will be in a custom function ?